httpd/handler_test.go

```diff
@@ -1327,6 +1327,86 @@ func TestHandler_serveWriteSeriesBoolValues(t *testing.T) {
 	}
 }
 
+func TestHandler_serveWriteSeriesBatch(t *testing.T) {
+	srvr := OpenAuthlessServer(NewMessagingClient())
+	srvr.CreateDatabase("foo")
+	srvr.CreateRetentionPolicy("foo", influxdb.NewRetentionPolicy("bar"))
+	srvr.SetDefaultRetentionPolicy("foo", "bar")
+
+	s := NewHTTPServer(srvr)
+	defer s.Close()
+
+	batch := `
+{
+    "database": "foo",
+    "retentionPolicy": "bar",
+    "points": [
+        {
+            "name": "disk",
+            "timestamp": "2009-11-10T23:00:00Z",
+            "tags": {
+                "host": "server01"
+            },
+            "values": {
+                "full": false
+            }
+        },
+        {
+            "name": "disk",
+            "timestamp": "2009-11-10T23:00:01Z",
+            "tags": {
+                "host": "server01"
+            },
+            "values": {
+                "full": true
+            }
+        },
+        {
+            "name": "disk",
+            "timestamp": "2009-11-10T23:00:02Z",
+            "tags": {
+                "host": "server02"
+            },
+            "values": {
+                "full_pct": 64
+            }
+        }
+    ]
+}
+`
+	status, body := MustHTTP("POST", s.URL+`/write`, nil, nil, batch)
+	if status != http.StatusOK {
+		t.Log(body)
+		t.Fatalf("unexpected status: %d", status)
+	}
+	time.Sleep(200 * time.Millisecond) // Ensure data node picks up write.
```

Comment:
Did we ever at an `/index` endpoint? It'd be nice to be able to wait by index instead of sleeping. I think this is the source of many issues on Travis.